src/plugins/intel_gpu/tests/functional/custom_op/custom_op_dynamic.cl

```diff
@@ -0,0 +1,10 @@
+__kernel void custom_add_kernel(
+    __global const INPUT0_TYPE* inp0,
+    __global OUTPUT0_TYPE* outp) {
+    int b = get_global_id(0);
+    int f = get_global_id(1);
+    int y = get_global_id(2);
+    // shape: [-1, 1, 2]
+    int id = b * 1 * 2 + f * 2 + y;
```

Comment:
Could you please use macro-defined sizes in this test kernel? Just to ensure they are correctly set for each dynamic case
Also, let's make the dynamic dimension not only the batch [dyn, static_dim1, dim2], but the innermost one as well, like this: [dyn, static_dim1, dyn]

Reply:
Updated.